src/storage.ts

```diff
@@ -7,24 +7,85 @@ import { createPool as MySQLCreatePool } from 'mysql2';
 import pg from 'pg';
 import Cursor from 'pg-cursor';
 
-const storageTypes = {
+enum StoreType {
+  DataStore = 1,
+  MessageStore,
+  EventLog,
+}
```

Comment:
you sure all good here? That only DataStore has an assigned value here

Reply:
I searched for how to make an enum in typescript and [this page](https://www.typescriptlang.org/docs/handbook/enums.html)  said that's how you do it, stating of the unassigned ones: "All of the following members are auto-incremented from that point on". Is this wrong?

Reply:
oh, no, that is not wrong. 
It's just looks like numbers(values) don't really matter here, so you could skip `= 1` as well, and it'd be autoassign to 0 and go +1 after that incrementally.

Completely up to you

Reply:
oh nifty, i will remove the number